source/Core/Drivers/BootLogo.cpp

```diff
@@ -35,9 +35,9 @@ void BootLogo::showNewFormat(const uint8_t *ptrLogoArea) {
 
   // Now draw in the frames
   int position = 1;
+  int len = (showNewFrame(ptrLogoArea + position));
```

Comment:
You cant pull it outside of the do-while loop like this.
You would want:

```
int len =0;
do {
len = (showNewFrame(ptrLogoArea + position));

```

Reply:
otherwise its going to only calculate len once 😓

Reply:
Will change this later.